src/cityreader/cityreader.py

```diff
@@ -14,16 +25,20 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
-cities = []
+cityList = []
+filename = open("cities.csv", "r")
+csvreader = csv.reader(filename)
+for row in csvreader:
+    cityList.append(row)
+cities = [[c[0], c[3], c[4]] for c in cityList if c[0] != "city"]
+
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
```

Comment:
Great stuff here Michael. You're bringing in many of the techniques from past lessons 👍 Did you experiment with adding your file reading logic above into the function? Might be able to lose a variable or two.